README.md

```diff
@@ -502,6 +511,38 @@ server:start()
 
 [socket_ref]: https://www.tarantool.io/en/doc/latest/reference/reference_lua/socket/#socket-tcp-server
 
+## Roles
+
+New roles could be accessed from this project.
+
+### `http_server` role
+
+This role can be used only with Tarantool 3.
+It allows configuring one or more HTTP servers.
+Those servers could be reused by several other roles.
+
+Example of the configuration:
+
+```yaml
+roles_cfg:
+  roles.http_server:
+    default:
+    - listen: 8081
+    additional:
+    - listen: '127.0.0.1:8082'
+```
+
+Server address should be provided either as a URI or as a single port
+(in this case, `0.0.0.0` address is used).
+
+User can access every working HTTP server from the configuration by name,
+using `get_server(name)` method.
+If the `name` argument is `nil`, the default server is returned
+(its name should be equal to constant `DEFAULT_SERVER_NAME`).
+
+For an example of using this role, refer to
+[this integration test](./test/integration/http_server_role_test.lua).
```

Comment:
Right now no other project supports this role. And to show an actual example, AFAIU, we either need to show how to support this role from the other role (which is done in `./test/integration/http_server_role_test.lua` via custom mock role), or just write some pulled from thin air example directly in `README`.

For instance, the second way will look like this:
`To show an example on how to use this role with other roles, lets consider role *my_role*, which uses HTTP servers for its own purposes. To configure the connection to each server, the config expects the name of the server and some endpoints to connect. In this example, config file may look like this:`
```yml
roles_cfg:
  roles.http_server:
    default:
    - listen: 8081
    additional:
    - listen: '127.0.0.1:8082'
  roles.my-role:
    - endpoints:
      - path: /metrics
      - path: /metrics/prometheus/
        format: prometheus
    - server: additional
      endpoints:
      - path: /metrics/
```
`Notice that the first connection will use the default server, since no name is provided.`

Or something like that. Will this be ok to put as the doc?

Reply:
I would like to see a code of minimal role that output "Hello world" + an example of the configuration for the role:
```yaml
roles_cfg:
  roles.http_server:
    default:
    - listen: 8081
    additional:
    - listen: '127.0.0.1:8082'
  roles.my-role:
    - httpd: additional
```